### PR TITLE
Error code for outdated drivers

### PR DESCRIPTION
**Pull request by summeroff**

### CODE REVIEW

**Review by pdamasceno — APPROVED** [review]

File: libobs/obs.h [hunk 1]
```diff
@@ -2111,6 +2111,9 @@ EXPORT void *obs_encoder_create_rerouted(obs_encoder_t *encoder,
 /** Returns whether encoder is paused */
 EXPORT bool obs_encoder_paused(const obs_encoder_t *output);
 
+/** Set encoder error to outputs */
+EXPORT void encoder_error_for_output(obs_encoder_t *encoder, const char * error_text);
```
Comment (EddyGharbi) [inline libobs/obs.h]: I would change the name of this function -> something describing more what it does, maybe `obs_outputs_set_last_error`?

Comment (summeroff) [reply libobs/obs.h]: done

File: libobs/obs-defs.h [hunk 2]
```diff
@@ -41,6 +41,8 @@
 #define OBS_OUTPUT_UNSUPPORTED -6
 #define OBS_OUTPUT_NO_SPACE -7
 #define OBS_OUTPUT_ENCODE_ERROR -8
+#define OBS_OUTPUT_SLOBS_IDS -64
```
Comment (EddyGharbi) [inline libobs/obs-defs.h]: Where/how is it used?

Comment (summeroff) [reply libobs/obs-defs.h]: Placeholder for error codes introduced for slobs. Same can be done by comment. But code more noticeable than comment.

**Review by EddyGharbi — APPROVED** [review]

**Comment by EddyGharbi** [comment]
Waiting on the tests to pass and I'll merge.

**Pull request merged**